### PR TITLE
all: updated code owners file

### PR DESCRIPTION
**Pull request by justelad**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,23 +9,24 @@ les/                            @zsfelfoldi
 light/                          @zsfelfoldi
 mobile/                         @karalabe
 p2p/                            @fjl @zsfelfoldi
```
Comment (zelig) [inline .github/CODEOWNERS]: p2p/simulations @lmars 
p2p/protocols @zelig

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -9,23 +9,24 @@ les/                            @zsfelfoldi
...
 swarm/dev                       @lmars
 swarm/fuse                      @jmozah @holisticode
 swarm/grafana_dashboards        @nonsense
 swarm/metrics                   @nonsense @holisticode
 swarm/multihash                 @nolash
-swarm/network/bitvector         @zelig @janos @gbalint
-swarm/network/priorityqueue     @zelig @janos @gbalint
+swarm/network/bitvector         @zelig @janos 
+swarm/network/priorityqueue     @zelig @janos 
 swarm/network/simulations       @zelig
```
Comment (zelig) [inline .github/CODEOWNERS]: warm/network/simulation @janos

File: .github/CODEOWNERS (hunk 1 above)
Comment (justelad) [reply .github/CODEOWNERS]: done

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -9,23 +9,26 @@ les/                            @zsfelfoldi
...
 swarm/grafana_dashboards        @nonsense
 swarm/metrics                   @nonsense @holisticode
 swarm/multihash                 @nolash
-swarm/network/bitvector         @zelig @janos @gbalint
-swarm/network/priorityqueue     @zelig @janos @gbalint
-swarm/network/simulations       @zelig
-swarm/network/stream            @janos @zelig @gbalint @holisticode @justelad
+swarm/network/bitvector         @zelig @janos 
+swarm/network/priorityqueue     @zelig @janos 
+swarm/network/simulations       @zelig @janos
```
Comment (zelig) [inline .github/CODEOWNERS]: `swarm/network/simulation @janos` to be added

**Pull request merged**